src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -622,80 +629,128 @@ private void ProcessGoAwayFrame(FrameHeader frameHeader)
             _incomingBuffer.Discard(frameHeader.Length);
         }
 
-        private async ValueTask AcquireWriteLockAsync()
+        private async ValueTask StartWriteAsync(int credit, CancellationToken cancellationToken = default, bool acquireWriteLock = true)
```

Comment:
It's confusing to call this parameter "credit" -- I originally assumed this had to do with window size management. Maybe call it "writeBytes" or something like that?

Reply:
Thanks. I've fixed this and plan to include the change in the next iteration.